### ISSUE TITLE
the accuracy is just about 50%

### ISSUE THREAD

**Issue by QinZibo** [moskomule/senet.pytorch#48]
I used the prertrained model, the command is 
`python -m torch.distributed.launch --nproc_per_node=8 imagenet.py /media/userdisk3/ImageNet_ILSVRC2012 --distributed --batch_size 640 --epochs 45`
,but after 30 epochs, the accuracy is still about 50%, just like the train stage at first epoch.

**Comment by moskomule** [comment]
I just confirmed the accuracy with the default setting.
